maintainers/maintainer-list.nix

```diff
@@ -5625,6 +5625,16 @@
     githubId = 510202;
     name = "Ismaël Bouya";
   };
+  impl = {
```

Comment:
`maintainer-list.nix` should be updated in a separate commit.

Do not open a pull request, only split this patch in a separate commit

https://stackoverflow.com/questions/6217156/break-a-previous-commit-into-multiple-commits